clients/java/src/test/java/io/camunda/client/OAuthCredentialsProviderTest.java

```diff
@@ -197,6 +198,28 @@ void shouldRequestTokenWithScopeAndAddToCall() throws IOException {
         .containsExactly(new Credential("Authorization", TOKEN_TYPE + " " + ACCESS_TOKEN));
   }
 
+  @Test
+  void shouldRequestTokenWithEmptyScopeAndAddToCall() throws IOException {
+    // given
+    final OAuthCredentialsProvider provider =
+        new OAuthCredentialsProviderBuilder()
+            .clientId(CLIENT_ID)
+            .clientSecret(SECRET)
+            .audience(AUDIENCE)
+            .scope("")
+            .authorizationServerUrl(tokenUrlString())
+            .credentialsCachePath(cacheFilePath.toString())
+            .build();
+    mockCredentials(ACCESS_TOKEN, "");
```

Comment:
💭  Initially I was wondering what the second parameter here means (just looking at it in github), but opening the IDE helped as it shows `mockCredentials(ACCESS_TOKEN, scope: "");`. Maybe it's beneficial to extract this into a `var scope = ""`, but given we are working with Java and everyone has a smart IDE this might just be fine.